charts/osm/values.yaml

```diff
@@ -159,3 +159,6 @@ OpenServiceMesh:
 
   # -- Run init container in privileged mode
   enablePrivilegedInitContainer: false
+
+  # -- Optional parameter to which Kubernetes distribution OSM is running on.
+  distribution: ""
```

Comment:
Purely a suggestion: For OpenShift, this is something we could detect automatically with something like `{{ Capabilities.APIVersions.Has "security.openshift.io/v1" }}`. I imagine this trick wouldn't work for all distributions, but maybe we could wrap this in a helper like we do for OSM namespace to default it to "openshift" when we can detect it.

Reply:
Using this suggestion. We can revisit having an explicit parameter if customers require it in the future.